src/types.js

```diff
@@ -0,0 +1,230 @@
+/**
+ * NOTE: These are temporary until Web5 JS can be converted to TypeScript and import the types from dwn-sdk-js.
+ */
+
+/**
+ * Web5 JS type definitions.
+ */
+
+/**
+ * @typedef {Object} Web5SendResponseMessage
+ * @property {ProtocolsConfigureDescriptor | ProtocolsQueryDescriptor | RecordsQueryDescriptor | RecordsReadDescriptor | RecordsWriteDescriptor} message
```

Comment:
a bit unfortunate that we have all these other types in a separate file instead of right next to where the actual types are declared/used

id hate to have to remember to also update this file whenever im changing something elsewhere in the repo :/

Reply:
Fair request although I kept the types together because they'll be gone once the lib is converted to TS and most of these types can be imported from `dwn-sdk-js`.  For now, I moved the two that are Web5 JS specific to be adjacent to the method that is returning this type since there is only one.  The rest will be removed in a future PR once types can be imported.